tests/Mutator/AbstractMutatorTestCase.php

```diff
@@ -66,7 +66,7 @@ protected function setUp(): void
         $this->mutator = $this->getMutator();
     }
 
-    public function doTest(string $inputCode, $expectedCode = null): void
+    public function doTest(string $inputCode, $expectedCode = null, $settings = []): void
```

Comment:
What about 
```suggestion
    public function doTest(string $inputCode, ?string $expectedCode = null, array $settings = []): void
```

Reply:
`?string $expectedCode` - we can't do it, since we use strings and arrays for 1 and multiple mutations cases correspondingly 

`array $settings` - done